src/js/script.js

```diff
@@ -105,6 +105,13 @@ $(function () {
             }
         });
     }
+    // News edited date
+    const newsEditedElements = document.querySelectorAll('.news-edited');
+    newsEditedElements.forEach(element => {
+    const editedDate = element.getAttribute('data-date');
+    const tooltipTitle = `${dayjs(editedDate).format('D MMMM YYYY MM:HH')}`;
```

Comment:
Please use `timestampToDate(timestamp, true)` function to format time.

Reply:
Done.